sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -455,22 +455,25 @@ def _extract_data_cb(self, messages: Any) -> Tuple[str, List[QueueMessage]]:
 class QueueProperties(DictMixin):
     """Queue Properties.
 
-    :keyword str name:
+    :keyword Optional[str] name:
         The name of the queue.
     :keyword Optional[Dict[str, str]] metadata:
         A dict containing name-value pairs associated with the queue as metadata.
         This var is set to None unless the include=metadata param was included
         for the list queues operation.
+    :keyword Optional[int] approximate_message_count:
```

Comment:
This isn't actually a keyword. The keyword is actually the header name 'x-ms-approximate-messages-count', though I don't think we should document that....